ocaml/xapi/xapi_host_driver.ml

```diff
@@ -249,12 +249,13 @@ let discover ~__context ~host =
     |> List.filter (fun (_, ds) -> ds <> [])
   with
   | Sys_error e ->
-      internal_error "Failed to parse drivers on host %s: %s"
-        (Ref.string_of host) e
+      D.debug "Failed to parse drivers on host %s: %s" (Ref.string_of host) e ;
```

Comment:
Maybe add a TODO here so that we know this needs to be changed once the foundation work is done?

Reply:
It's in a separate temporary commit that will just be dropped.